components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -614,27 +612,23 @@ public void removeTempAuthnContextData(String key, String type) {
     }
 
     private void setBlobObject(PreparedStatement prepStmt, Object value, int index)
-            throws SQLException, IOException {
+            throws SQLException, IOException, SessionSerializerException {
         if (value != null) {
-            ByteArrayOutputStream baos = new ByteArrayOutputStream();
-            ObjectOutputStream oos = new ObjectOutputStream(baos);
-            oos.writeObject(value);
-            oos.flush();
-            oos.close();
-            InputStream inputStream = new ByteArrayInputStream(baos.toByteArray());
+            InputStream inputStream = FrameworkServiceDataHolder.getInstance().
```

Comment:
`FrameworkServiceDataHolder.getInstance().getSessionSerializer()` can lead to null.

Reply:
https://github.com/wso2/carbon-identity-framework/pull/3631/files#diff-82b398c13cf564351e870b3eefb7d9c6916ffb827579298085c0c4769aef6636R276 Added these serializer set if its null in session data store constructor

Reply:
It's possible to get https://github.com/wso2/carbon-identity-framework/pull/3631/files#diff-8bccdb72ac4633653cb8821508b1c64219f85e428f8bf61fe51c7540c8946b4fR494 called after calling the https://github.com/wso2/carbon-identity-framework/pull/3631/files#diff-82b398c13cf564351e870b3eefb7d9c6916ffb827579298085c0c4769aef6636R276.

Reply:
https://github.com/wso2/carbon-identity-framework/pull/3631/files#diff-8bccdb72ac4633653cb8821508b1c64219f85e428f8bf61fe51c7540c8946b4fR490 Added default serializer in the unset method